lib/wasi-tests/build/wasitests.rs

```diff
@@ -83,7 +83,7 @@ pub fn compile(file: &str, ignores: &HashSet<String>) -> Option<String> {
         .arg("+nightly")
         .arg("--target=wasm32-wasi")
         .arg("-C")
-        .arg("opt-level=s")
+        .arg("opt-level=z")
```

Comment:
This seems like it's part of a different change?

Reply:
It is, but using a new version of nightly locally means all the generated wasm modules will change anyways, so I figured I might as well make this change that I've been meaning to make for a while anyways